packaging/package.csproj

```diff
@@ -34,6 +34,16 @@
         <PackageReference Include="System.CommandLine" PrivateAssets="all" IncludeAssets="runtime"/>
     </ItemGroup>
 
+    <Target Name="PreparePackageReleaseNotesFromFile" BeforeTargets="GenerateNuspec">
+        <ReadLinesFromFile File="../docs/ReleaseNotes/v0.2.0.md" >
```

Comment:
Where does ReleaseNoteLines get consumed?

Reply:
ah, that was an old version of this - I need to remove it since I'm using the other method of reading the file.

Reply:
fixed!